app.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express')
+const app = express()
+const path = require('path')
+const PORT = process.env.PORT || 5000
+
+app.use(express.static('public'))
+
+app.get('/index/', (request, response)=>{
+    response.sendFile(path.resolve('./public/index.html'))
+})
+
+app.get('/post/:id', (request, response) =>{
+    response.sendFile(path.resolve('./public/post.html'))
+} )
+
+app.get('/api/:data', (req, res) => {
+    res.sendFile(path.resolve('./public/' + req.params.data))
```

Comment:
Your api endpoint doesn't live inside `public` it's inside `api` so you would need to use `'./api/' + req.params.data` for the file path to your custom endpoint.